pages/glossary/matching-characters.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Matching characters
+key: matching-characters
+---
+
+A sequence of [characters](#character) is considered to match another if, after removing leading and trailing [space characters](https://www.w3.org/TR/html/infrastructure.html#space-characters) and replacing remaining occurrences of one or more space characters with a single space, the two sequences of characters are equal character by character, ignoring any differences in letter casing.
```

Comment:
I can't find a glossary entry for #character. Did you mean #encoded-character?

Reply:
The issue was already raised as #783. Last Tuesday. @WilcoFiers said that we don't really need a definition of "character" (and I removed the link from "matching characters"). I'll clean up that link too.

Reply:
> @WilcoFiers said that we don't really need a definition of "character"

Let me for the record state that I had a chat with @WilcoFiers about this and hopefully convinced him that the term "character" is _extremely_ ambiguous unless properly defined.